components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -505,6 +505,26 @@ - (void)testBottomDrawerScrollingEnabled {
   XCTAssertEqual(self.fakeScrollView.scrollEnabled, NO);
 }
 
+- (void)testSetTrackingScrollView {
+  // Given
+  MDCBottomDrawerPresentationController *drawerPresentationController = nil;
+  if ([self.drawerViewController.presentationController
+          isKindOfClass:[MDCBottomDrawerPresentationController class]]) {
+    drawerPresentationController =
+        (MDCBottomDrawerPresentationController *)self.drawerViewController.presentationController;
+  } else {
+    drawerPresentationController = nil;
```

Comment:
Tip: You can delete the "else" since it does nothing.

Reply:
Done